compiler/z/codegen/BinaryCommutativeAnalyser.cpp

```diff
@@ -245,8 +245,12 @@ TR_S390BinaryCommutativeAnalyser::genericAnalyser(TR::Node * root, TR::InstOpCod
 
    bool isLoadNodeNested = false;
 
+   // All of the memToRegOpCodes (MGH, MSGC and MSC) used in the if block below
+   // rely on the miscellaneous-instruction-extension facility 2 being installed
+   const bool supportsMIE2 = cg()->comp()->target().cpu.getSupportsMiscellaneousInstructionExtensions2Facility();
+
    // TODO: add MH and MHY here; outside of the z14 if check.
-   if(cg()->comp()->target().cpu.getSupportsArch(TR::CPU::z14))
+   if(supportsMIE2)
```

Comment:
This variable has only one use now. Can it be folded into the `if`?

Reply:
Fixed in commit beec7ce